about.md

```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
+## About the MDAnalysis Project
```

Comment:
Should this be changed to "Mission" ?

Repeating "About" in two headings looks a bit awkward.

Reply:
```suggestion
## Mission
```

Reply:
I like the change